### PR TITLE
添加杭州滨江星巴克（星光大道店）

### PR DESCRIPTION
**Pull request by makohill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@
 
 ## 城市列表
 
-* [北京 (8)](beijing.geojson)
+* [北京 (7)](beijing.geojson)
```
Comment (crispgm) [inline README.md]: Why the number decreased? I think it is a mistake. :)

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 ### 通过手机 App
 
-推荐使用 [Speedtest](http://www.speedtest.net/mobile/)（[iOS](https://itunes.apple.com/app/speedtest-net-mobile-speed/id300704847?mt=8) / [Android](https://play.google.com/store/apps/details?id=org.zwanoo.android.speedtest)），既可测试网络速度也能获取当前经纬度。
+推荐使用 [Speedtest](http://www.speedtest.net/mobile/) ([iOS](https://itunes.apple.com/app/speedtest-net-mobile-speed/id300704847?mt=8)/[Android](https://play.google.com/store/apps/details?id=org.zwanoo.android.speedtest))，既可以测试网络速度也可以获取当前经纬度。
```
Comment (xhacker) [inline README.md]: I think this is also changed by accident? Can you also revert this?

**Comment by xavierchow** [comment]
Can you squash the git commits, otherwise LGTM.

**Comment by crispgm** [comment]
@makohill Have a tip for you.

After `git rebase`, use `git push -f` instead. Otherwise, you have to merge and get an extra commit like "	Merge branch 'master' of https://github.com/makohill/awesome-cn-cafe"

**Comment by xhacker** [comment]
Hey @makohill, do you want to make a new pull request?

**Pull request closed without merging**